packages/admin/src/FilamentManager.php

```diff
@@ -181,6 +182,26 @@ public function getResources(): array
         return array_unique($this->resources);
     }
 
+    public function getResource(Model | string $model): ?Resource
+    {
+        $modelFqcn = ($model instanceof Model)
+            ? $model::class
+            : $model;
+
+        foreach ($this->getResources() as $resourceClass) {
+            /**
+             * @var Resource $instance
+             */
+            $instance = new $resourceClass();
+
+            if ($modelFqcn === $instance::getModel()) {
+                return new $instance;
```

Comment:
`$instance` is already an instance of the resource class. Just return the class string directly, the user can instantiate the object if they want to.

Reply:
Oh god, little mistake... I fixed it. 

I want return `$instance` and not `$resourceClass` because I want keep completion and be sure to manipulate a `Resource` and not a string (fqcn)